{
  "cve_id": "CVE-2021-4070",
  "github_data": {
    "repository": "v2fly/v2ray-core",
    "fix_commit": "c1af2bfd7aa59a4482aa7f6ec4b9208c1d350b5c",
    "related_commits": [
      "c1af2bfd7aa59a4482aa7f6ec4b9208c1d350b5c",
      "c1af2bfd7aa59a4482aa7f6ec4b9208c1d350b5c"
    ],
    "patch_url": null
  },
  "vulnerability_details": {
    "cvss_score": 9.1,
    "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H",
    "cwe_id": "CWE-193",
    "description": "Off-by-one Error in GitHub repository v2fly/v2ray-core prior to 4.44.0.",
    "attack_vector": "NETWORK",
    "attack_complexity": "LOW"
  },
  "temporal_data": {
    "published_date": "2022-02-23T20:15:09.167",
    "last_modified": "2024-11-21T06:36:50.743",
    "fix_date": null
  },
  "references": [
    {
      "url": "https://github.com/v2fly/v2ray-core/commit/c1af2bfd7aa59a4482aa7f6ec4b9208c1d350b5c",
      "source": "security@huntr.dev",
      "tags": [
        "Patch",
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://huntr.dev/bounties/8da19456-4d89-41ef-9781-a41efd6a1877",
      "source": "security@huntr.dev",
      "tags": [
        "Exploit",
        "Patch",
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://github.com/v2fly/v2ray-core/commit/c1af2bfd7aa59a4482aa7f6ec4b9208c1d350b5c",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Patch",
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://huntr.dev/bounties/8da19456-4d89-41ef-9781-a41efd6a1877",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Exploit",
        "Patch",
        "Third Party Advisory"
      ]
    }
  ],
  "collection_metadata": {
    "collected_at": "2025-01-11T23:02:59.743401",
    "processing_status": "raw"
  }
}